seqr/migrations/0081_savedvariant_gene_ids.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 4.2.24 on 2025-09-25 19:26
+
+from collections import defaultdict
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+from clickhouse_search.backend.functions import ArrayMap
+from clickhouse_search.search import get_annotations_queryset
+
+BATCH_SIZE = 10000
+
+class JsonKeys(models.Func):
+    function = 'jsonb_object_keys'
+
+
+def populate_variant_gene_ids(apps, schema_editor):
```

Comment:
this runs in about a minute so should not be a problem to run as a normal migration